src/main/java/com/microsoft/graph/http/GraphServiceException.java

```diff
@@ -387,7 +391,12 @@ public GraphError getServiceError() {
         }
         GraphErrorResponse error;
         try {
-            error = serializer.deserializeObject(rawOutput, GraphErrorResponse.class, responseHeadersHelper.getResponseHeadersAsMapOfStringList(response));
+            // we need a "copy" of the stream, so we can log the raw output if it cannot be parsed
+            error = serializer.deserializeObject(
+                    new ByteArrayInputStream(rawOutput.getBytes(UTF_8)),
```

Comment:
shouldn't this stream be closed after deserialization?

Reply:
Not really, because close() on a ByteArrayInputStream has no effect.

Reply:
resolving to the benefit of the other convo